diff --git a/node_modules/expo-router/build/ui/TabSlot.js b/node_modules/expo-router/build/ui/TabSlot.js
index 345c422..f975fab 100644
--- a/node_modules/expo-router/build/ui/TabSlot.js
+++ b/node_modules/expo-router/build/ui/TabSlot.js
@@ -71,6 +71,7 @@ const styles = react_native_1.StyleSheet.create({
     screenContainer: {
         flexShrink: 0,
         flexGrow: 1,
+        height: '100%',
     },
     focused: {
         zIndex: 1,
